content/manager_webui/custom-widgets.md

```diff
@@ -98,12 +98,30 @@ Option                 | Type    | Default | Description
 `categories`           | array   | -       | This property specifies in which categories this widget shall be visible. It may take an array containing one or more of the values defined in `Stage.GenericConfig.CATEGORY` object: `BLUEPRINTS` ('Blueprints' category), `DEPLOYMENTS` ('Deployments'), `BUTTONS_AND_FILTERS` ('Buttons and Filters'), `CHARTS_AND_STATISTICS` ('Charts and Statistics'), `EXECUTIONS_NODES` ('Executions/Nodes'), `SYSTEM_RESOURCES` ('System Resources'), `OTHERS` ('Others'), `ALL` ('All'). Optional.
 `permission`           | string  | -       | This property specifies which user may later access and view this widget. It may take one of the following three values defined in `Stage.GenericConfig.CUSTOM_WIDGET_PERMISSIONS` object: `CUSTOM_ADMIN_ONLY` (applies for 'sys_admin' and 'manager' roles), `CUSTOM_SYS_ADMIN_ONLY` (applies for 'sys_admin' only, `CUSTOM_ALL` (applies to all user-roles). Mandatory.
 
-**Note:** initialConfiguration supports 4 generic pre-made config fields (see `fetchUrl` for example):
+#### initialConfiguration
+`initialConfiguration` supports 4 generic pre-made config fields (see `fetchUrl` for example):
+
 * `Stage.GenericConfig.POLLING_TIME_CONFIG(int)` - How often to refresh the data (in seconds)
 * `Stage.GenericConfig.PAGE_SIZE_CONFIG()` - Takes no arguments and set's the page size to default 5
 * `Stage.GenericConfig.SORT_COLUMN_CONFIG(string)` - Column name to sort by
 * `Stage.GenericConfig.SORT_ASCENDING_CONFIG(boolean)` - Change sorting order (true=ascending)
 
+Despite listed above, you can create your own configuration fields. Example of configuration with user-defined fields:
```

Comment:
I think maybe "despite" is not the exact word to use here...maybe "In addition"?

Reply:
You're right. Fixed.